VoiceRepeater/readme.md

```diff
@@ -9,3 +9,7 @@
 ### And plays it!
 
 ### Requirements: Python, SpeechRecognition and playsound
+
+
+
+I am just trying to contribute please let me do this.....thank you :)
```

Comment:
# At least comment it.
# This is not a repository for practice purposes of things like this.
# There are other good repositories available for Pull Request practices.
# Please make this program or revert the changes.